cas-server-core/src/main/java/org/jasig/cas/CentralAuthenticationServiceImpl.java

```diff
@@ -328,7 +316,8 @@ public ServiceTicket grantServiceTicket(
                 this.serviceTicketExpirationPolicy,
                 currentAuthentication != null);
 
-        this.serviceTicketRegistry.addTicket(serviceTicket);
+        this.ticketRegistry.updateTicket(ticketGrantingTicket);

```

Comment:
Indeed, if we have a new ST, the TGT must be updated, but I thought it was internally done by the `AbstractDistributedRegistry` by using the delegators: https://github.com/Jasig/cas/blob/master/cas-server-core/src/main/java/org/jasig/cas/ticket/registry/AbstractDistributedTicketRegistry.java#L230. Am I missing the point?